roles/ceph-grafana/tasks/configure_grafana.yml

```diff
@@ -1,15 +1,4 @@
 ---
-- name: install ceph-grafana-dashboards package on RedHat or SUSE
```

Comment:
in fact I'm wondering how this will work in non containerized deployment because the mgr node doesn't have the ceph dashboard package installed so you won't be able to run the dashboard update command.

So I guess there's 2 solutions:
  - keep the `ceph-grafana-dashboards` package installation on the grafana node and not running the dashboard update command.
  - install `ceph-grafana-dashboards` package on the mgr node and run the dashboard update command.

wdyt ? Maybe I'm missing something here